src/system-component-workspace/system-component.service.ts

```diff
@@ -28,6 +28,9 @@ export class SystemComponentWorkspaceService {
     private readonly mpService: MonitorPlanWorkspaceService,
   ) {}
 
+  // log values in terminal for debugging purpose
+  private readonly logger = new Logger(SystemComponentWorkspaceService.name);
```

Comment:
This is usually added in the constructor, I believe NestJS then can make use of a singleton instance:
```suggestion
    private readonly mpService: MonitorPlanWorkspaceService,
    private readonly logger: Logger,
  ) {
    this.logger.setContext(SystemComponentWorkspaceService.name);
  }
```